src/pages/[CompanyID].js

```diff
@@ -0,0 +1,175 @@
+import {ErrorMessage} from '@hookform/error-message';
+import {useRouter} from 'next/router';
+import {useForm} from 'react-hook-form';
+
+import StyledForm from '../components/Forms/StyledComponents/styledForm';
+import StyledInput from '../components/Forms/StyledComponents/styledInput';
+import StyledLabel from '../components/Forms/StyledComponents/styledLabel';
+import StyledSubmitButton from '../components/Forms/StyledComponents/styledSubmitButton';
+import StyledWrapper from '../components/styledClientWrapper';
+import useClientStore from '../hooks/useClientStore';
+
+export default function EditClient() {
+	const {
+		register,
+		formState: {errors},
+		handleSubmit,
+	} = useForm({
+		criteriaMode: 'all',
+	});
+	const onSubmit = data => {
+		wantedClient.CompanyAdress = data.Adress;
+		wantedClient.CompanyCity = data.city;
+		wantedClient.CompanyName = data.clientName;
+		wantedClient.CompanyTaxID = data.taxId;
+		wantedClient.CompanyZipCode = data.postalCode;
+		router.push({
+			pathname: '/',
+		});
+	};
+	const router = useRouter();
+	const ref = router.query;
+	const allClients = useClientStore(state => state.clients);
+	const ClientsArray = allClients.filter(ele => {
```

Comment:
Did I miss this variable during the last RP update? 🫣
**Please don't use PascalCase rather camelCase.** 
  
```suggestion
	const clientsArray = allClients.filter(ele => {
```